src/SignalR/clients/ts/FunctionalTests/SignalR.Npm.FunctionalTests.npmproj

```diff
@@ -39,11 +36,23 @@
       <_TestSauceArgs>--verbose --no-color --configuration $(Configuration) --sauce-user "$(SauceUser)" --sauce-key "$(SauceKey)"</_TestSauceArgs>
       <_TestSauceArgs Condition="'$(BrowserTestHostName)' != ''">$(_TestSauceArgs) --use-hostname "$(BrowserTestHostName)"</_TestSauceArgs>
     </PropertyGroup>
+    
+    <Message Condition="'$(DailyTests)' == 'true'" Text="Running JavaScript client Browser tests on SauceLabs" Importance="high" />
     <Message Text="test:sauce Args = $(_TestSauceArgs)" Importance="high" />
     <Exec Command="npm run test:sauce -- $(_TestSauceArgs)"
           Condition="'$(DailyTests)' == 'true'"
+          ContinueOnError="true"
           WorkingDirectory="$(RepositoryRoot)src/SignalR/clients/ts/FunctionalTests"
-          IgnoreStandardErrorWarningFormat="true" />
+          IgnoreStandardErrorWarningFormat="true">
+      <Output TaskParameter="ExitCode" PropertyName="ErrorCode"/>
+    </Exec>
+    <Message Importance="high" Text="$(ErrorCode)"/>
+
+    <Message Condition="'$(ErrorCode)' == '1'" Text="First run failed with $(ErrorCode) -- Rerunning sauce labs tests" Importance="high" />
+    <Exec Command="npm run test:sauce -- $(_TestSauceArgs)"
```

Comment:
So this will only retry a maximum of once? Also would it make sense to try again if ErrorCode is anything other than 0?